jdaviz/configs/cubeviz/helper.py

```diff
@@ -123,8 +123,8 @@ def specviz(self):
     def get_data(self, data_label=None, spatial_subset=None, spectral_subset=None, function=None,
                  cls=None):
         """
-        Returns data with name equal to data_label of type cls with subsets applied from
-        subset_to_apply.
+        Returns data with name equal to ``data_label`` of type ``cls`` with subsets applied from
+        ``spatial_subset`` and/or ``spectral_subset`` using ``function`` if applicable.
```

Comment:
Maybe say where this is applicable?

Reply:
I expand on that more in the parameters section, is there something specifically you want to see here?

Reply:
Ah I see, missed it since it wasn't in the diff. This looks fine 👍